libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxNavigationTest.kt

```diff
@@ -141,6 +147,17 @@ class MapboxNavigationTest {
                 locationEngine,
                 locationEngineRequest
             )
+        rerouteController = mockk(relaxed = true, relaxUnitFun = true)
```

Comment:
```suggestion
        rerouteController = mockk(relaxUnitFun = true)
```
Let's handle the return types explicitly if needed.